.circleci/config.yml

```diff
@@ -590,6 +590,7 @@ workflows:
             branches:
               only:
                 - master
+                - feat_pdef_tunable
```

Comment:
don't forget to remove this

Reply:
Yup - I need test images in the meantime ;)